ganga/GangaCore/Runtime/GPIFunctions.py

```diff
@@ -93,3 +95,9 @@ def convert_merger_to_postprocessor(j):
         mp.process_objects.append(stripProxy(j).merger)
         stripProxy(j).postprocessors = mp
 
+def runfile(path_to_file):
+    """ A wrapper for the runpy.run_path() function. """
```

Comment:
I suggest adding an example of usage here as well. Maybe just the simple usage `runpy('myfile.py')` but then also add an example of what can be in `myfile.py`. The important thing to illustrate is how it behaves with the namespace of the `Ganga` session. So can I have 

```
j = Job()
j.submit()
```
in the file? And comment on if the object `j` in this case will be visible at the main prompt after running.